sdks/java/io/splunk/src/main/java/org/apache/beam/sdk/io/splunk/SplunkIO.java

```diff
@@ -264,6 +269,35 @@ public Write withDisableCertificateValidation(Boolean disableCertificateValidati
           .build();
     }
 
+    /**
+     * Same as {@link Builder#withRootCaCertificatePath(ValueProvider)} but without a {@link
+     * ValueProvider}.
+     *
+     * @param rootCaCertificatePath Path to root CA certificate
+     * @return {@link Builder}
+     */
+    public Write withRootCaCertificatePath(ValueProvider<String> rootCaCertificatePath) {
+      checkArgument(
```

Comment:
Please use CheckNotNull instead: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/d8c84c6f735848c715eef23c1af3486a1248f11c/v1/src/main/java/com/google/cloud/teleport/splunk/SplunkEvent.java#L78

Reply:
Done (along with the other parameters in this Builder)